libdnf/dnf-sack.cpp

```diff
@@ -2335,20 +2335,29 @@ setModuleExcludes(DnfSack *sack, const char ** hotfixRepos,
 {
     dnf_sack_set_module_excludes(sack, nullptr);
     std::vector<std::string> names;
+    std::vector<std::string> srcNames;
     libdnf::DependencyContainer nameDependencies{sack};
     libdnf::Nevra nevra;
     for (const auto &rpm : includeNEVRAs) {
         if (nevra.parse(rpm.c_str(), HY_FORM_NEVRA)) {
-            names.push_back(nevra.getName());
-            nameDependencies.addReldep(nevra.getName().c_str());
+            auto arch = nevra.getArch();
+            // src packages do not provide nothing and should not trigger removal of binar packages
```

Comment:
replace with: source packages do not provide anything and must not cause excluding binary packages